examples/test_azure_data_exfiltration_protection/firewall.tf

```diff
@@ -1,13 +1,3 @@
-locals {
```

Comment:
@nkvuong , can you give this a quick review? We will plan to set the major version to 1.0.0